pysph/tools/geometry.py

```diff
@@ -165,9 +165,9 @@ def rotate(x, y, z, axis=np.array([0.0, 0.0, 1.0]), angle=90.0):
     >>>z = np.array([0.5, -1.5, 0.0])
     >>>axis = np.array([0.0, 0.0, 1.0])
     >>>rotate(x, y, z, axis, 90.0)
-    (array([ 0.29212042, -0.5, 2.31181936]),
-     array([ -0.5, 3.31047738, 11.12095476]),
-     array([-0.5, -0.5, 3.5]))
+    (array([ 1.00000000e+00,  4.25628483e-17, -1.50000000e+00]),
```

Comment:
Curious, is this a mistake in the documentation?

Reply:
Yes.